lib/camp3/authorization.rb

```diff
@@ -59,8 +59,8 @@ def update_access_token!(refresh_token = nil)
     private
 
     def store_tokens(token)
-      Camp3.access_token = token.access_token
-      Camp3.refresh_token = token.refresh_token
+      @access_token = token.access_token
```

Comment:
Required so that the same client can reuse the new access token. Otherwise, it will fails because it will be still using the old access token